Formula/php@8.0.rb

```diff
@@ -9,7 +9,7 @@ class PhpAT80 < Formula
 
   livecheck do
     url "https://www.php.net/downloads"
-    regex(/href=.*?php[._-]v?(\d+(?:\.\d+)+)\.t/i)
+    regex(/href=.*?php[._-]v?(#{Regexp.escape(version.major_minor)}(?:\.\d+)+)\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?php[._-]v?(#{Regexp.escape(version.major_minor)}(?:\.\d+)*)\.t/i)
```

Just one small tweak to align this with the regex used in `php@7.4` as well as other regexes that target a specific major/minor.

Reply:
Works for me! 👍